packages/solana/src/connectors/walletConnectConnector.ts

```diff
@@ -51,7 +52,7 @@ export class WalletConnectConnector extends BaseConnector implements Connector {
 
     UniversalProviderFactory.getProvider().then(provider => {
       provider.on('session_delete', () => {
-        delete provider.session?.namespaces['solana']
+        delete provider.session?.namespaces[ConstantsUtil.CHAIN.SOLANA]
```

Comment:
I guess we could revert these because the purpose of the `provider.session?.namespaces['solana']` is not the same why we are using the `Chain` type. Let's keep it same for now. There should be another enum for namespaces.

Reply:
Let's keep it. Using constants rather than hard coded a better way